Code/GraphMol/FileParsers/CDXMLParser.cpp

```diff
@@ -38,6 +38,8 @@ const std::string CDX_ATOM_ID("_CDX_ATOM_ID");
 const std::string CDX_BOND_ID("_CDX_BOND_ID");
 const std::string CDX_BOND_ORDERING("CDXML_BOND_ORDERING");
 
+const double RDKIT_DEPICT_BONDLENGTH = 1.5;
```

Comment:
might as well use a constexpr for this
```suggestion
constexpr double RDKIT_DEPICT_BONDLENGTH = 1.5;
```